keadm/cmd/keadm/app/cmd/edge/batchjoin.go

```diff
@@ -0,0 +1,179 @@
+package edge
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sync"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh"
+	"gopkg.in/yaml.v3"
+)
+
+type Node struct {
+	IP              string `yaml:"ip"`
+	EdgeNodeName    string `yaml:"EdgeNodeName"`
+	Architecture    string `yaml:"architecture"`
```

Comment:
Can we detect this from the system that is going to join the cluster, instead of having this parameter to user burden?

```bash
tomoyafujita@~/Downloads/temp >dpkg-architecture -q DEB_BUILD_ARCH
amd64

tomoyafujita@~/Downloads/temp >dpkg --print-architecture
amd64

ubuntu@raspi4-1:~$ dpkg-architecture -q DEB_BUILD_ARCH
arm64

ubuntu@raspi4-1:~$ dpkg --print-architecture
arm64

```